DragonSlayer.h

```diff
@@ -1,8 +1,19 @@
 #pragma once
 
+#include "Character.h"
+#include "AttackItem.h"
+
 struct DragonSlayer : Character
 {
+    DragonSlayer(const std::string, int, int);
+
+    const std::string& getName() override;
+    std::string getStats() override;
     void attack(Character& other) override;
 private:
     const std::string name;
-};
\ No newline at end of file
+    AttackItem attackItem;
```

Comment:
Look at how the other items are used in Character.h/cpp.   They can only be used once, so they are reset after use. 
- [ ] wrap AttackItem in a unique_ptr so it can be reset after use.